controllers/operands/kubevirt.go

```diff
@@ -307,6 +380,13 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 
 	obsoleteCPUs, minCPUModel := getObsoleteCPUConfig(hc.Spec.ObsoleteCPUs)
 
+	if client != nil {
+		rateLimiter, err = hcoTuning2Kv(ctx, client, &hc.Spec.TuningPolicy, hcNamespace)
```

Comment:
Not sure about returning error here. Unlike other cases here, this error is not coming from a wrong configuration of the  HyperConverged CR, but from the CM. I don't want to kill the reconcile if the CM is not there (yet). @tiraboschi, what do you think?

Reply:
I think you are right and we can just throw a warning to inform the user that something happened and remove the `tuningPolicy` from the HCO spec. Just to not confuse users if they check the hco definition and they see that the `tuningPolicy` is present but the rates are not being configured (even if we notify them that the CM is missing or missconfigured and therefore this is not going to take effect).

Reply:
I don't want to edit the HyperConverged CR, if possible.

Maybe we need to catch it in the webhook. But this will only solve the creating or editing the HC CR use cases. But it won't help for the case of deleting or wrongly editing the CM, so I'm not sure.

Reply:
OK, let's do the following:
If the policy is "configmap" and the CM is missing or if its content is wrong, we'll write a warning to the log + trigger an event to notify that. @tiraboschi - we can also add a new metric (more general one: wrong config or something) + an alert.

In this case, HCO will ignore the policy field. This should be well documented.

Reply:
Addressed.